mcp_runtime_template_hg/api/app.py

```
@@ -1,6 +1,8 @@
 from fastapi import FastAPI
+
 app = FastAPI()
 
-@app.post('/mcp/agent/run')
+
+@app.post("/mcp/agent/run")
 def run_agent(payload: dict):
```

Ruff (0.11.9)

Missing return type annotation for public function `run_agent`

(ANN201)